### PR TITLE
RES-534 Rest in peace respose coverter not handling nil class

### PR DESCRIPTION
**Pull request by mur-wtag**
[RES-534](https://welltravel.atlassian.net/browse/RES-534)

### CODE REVIEW

File: lib/rest_in_peace/response_converter.rb [hunk 1]
```diff
@@ -28,6 +29,8 @@ def result
         convert_from_hash
       when 'String'
         body
+      when 'NilClass'
+        { body: body, status: status }
```
Comment (luxflux) [inline lib/rest_in_peace/response_converter.rb]: Can't we just return `nil` here? Returning a hash means (implicitlty) that there was content that had to be parsed.

Comment (mur-wtag) [inline lib/rest_in_peace/response_converter.rb]: so returning just `body`, okay?

Comment (luxflux) [inline lib/rest_in_peace/response_converter.rb]: I'd return `nil` explicitly.

File: lib/rest_in_peace/response_converter.rb [hunk 2]
```diff
@@ -28,6 +28,8 @@ def result
         convert_from_hash
       when 'String'
         body
+      when 'NilClass'
```
Comment (mur-wtag) [inline lib/rest_in_peace/response_converter.rb]: @luxflux Also please consider this comment https://github.com/wtag/reservationmanagement-client-ruby/blob/RES-528-remove-reservation-from-queue/spec/reservationmanagement/client/queue_spec.rb#l53-55

**Comment by luxflux** [comment]
Looks good, please open a PR on upstream.

**Pull request closed without merging**